ocean_lib/models/arguments.py

```diff
@@ -18,6 +20,28 @@
 logger = logging.getLogger("ocean")
 
 
+class FeeTokenArguments:
+    def __init__(
+        self,
+        address: Optional[str] = None,
+        token: Optional[str] = None,
+        amount: Optional[int] = 0,
+    ):
+        self.address = (
+            Web3.toChecksumAddress(address.lower()) if address else ZERO_ADDRESS
+        )
+        self.token = Web3.toChecksumAddress(token.lower()) if token else ZERO_ADDRESS
+
+        self.amount = amount
+
+    def to_tuple(self):
+        return (
```

Comment:
Why does it need to apply lower() and checksum() if it's already stored as these?